scheduler/config-k8s.edn

```diff
@@ -6,7 +6,7 @@
  :authorization-config {;; These users have admin privileges when using configfile-admins-auth;
                         ;; e.g., they can view and modify other users' jobs.
                         ;; Sets "admin" and the user defined in "COOK_ONE_USER_AUTH" to be admins for local development.
-                        :admins #{"admin" #config/env "COOK_ONE_USER_AUTH"}
+                        :admins #{"admin" "root" #config/env "COOK_ONE_USER_AUTH"}
```

Comment:
Why is this changing?

Reply:
This is a leak for my run-locally changes that was applied on top of the patch. Removed that commit from the push.